medcat/meta_cat.py

```diff
@@ -77,16 +77,31 @@ def get_model(self, embeddings: Optional[Tensor]) -> nn.Module:
                 The embedding densor
 
         Raises:
-            ValueError: If the meta model is not LSTM
+            ValueError: If the meta model is not LSTM or BERT
 
         Returns:
             nn.Module:
                 The module
         """
         config = self.config
+        from medcat.utils.meta_cat.models import LSTM
```

Comment:
Do we need to import it here?
I think the idea previously was that this would only get imported run time if/when needed.

Reply:
Yes I had it inside the if else statements, but since we need to declare the type for 'model', we need to import both of them `(Line 90)`

Reply:
You should be able to just use the common base class (`nn.Module`) for type annotation.